app/src/main/java/fr/free/nrw/commons/nearby/NearbyFragment.java

```diff
@@ -135,14 +135,15 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     /**
      * Hide or expand bottom sheet according to states of all sheets
      */
-    public void listOptionMenuIteClicked() {
+    public boolean listOptionMenuIteClicked() {
```

Comment:
Rather than change this method, maybe create a method here called something like `isBottomSheetExpanded()`
Probably just a one-liner like: `return bottomSheetBehavior.getState() == BottomSheetBehavior.STATE_EXPANDED`

Reply:
@domdomegg the function also changes the state apart from returning the current state. Do you mean make this function one liner and change the state from where the function was called?

Reply:
I mean undo the changes to `listOptionMenuIteClicked`, and create a new one-liner method in `NearbyFragment` which just returns the state (as a boolean) of the bottom sheet. Then we can change MainActivity line 272 to something like:

```
NearbyFragment nearbyFragment = (NearbyFragment) contributionsActivityPagerAdapter.getItem(1);

if(nearbyFragment.isBottomSheetExpanded()) {
    nearbyFragment.listOptionMenuIteClicked();
} else {
    viewPager.setCurrentItem(0);
}
```

Also in another PR it might be worth renaming `listOptionMenuIteClicked` to something like `toggleBottomSheet`

Reply:
BTW once done please re-request a review from me and I can get this merged :)